comtypes/tools/codegenerator.py

```diff
@@ -345,6 +346,11 @@ def ArrayType(self, tp):
     _enumvalues = 0
     def EnumValue(self, tp):
         value = int(tp.value)
+        if keyword.iskeyword(tp.name):
+            # XXX use logging!
+            if __debug__:
```

Comment:
It doesn't look like there's any logging already in this file, so I'd probably just leave it out. Or you could put the print statement in but commented out, so someone could uncomment it to debug.